corehq/apps/userreports/reports/builder/const.py

```diff
@@ -28,3 +28,9 @@
 PROPERTY_TYPE_CASE_PROP = "case_property"
 PROPERTY_TYPE_QUESTION = "question"
 PROPERTY_TYPE_RAW = "raw"
+
+
+# note: these values are duplicated in js/constants.js
+PRE_FILTER_VALUE_IS_EMPTY = 'Is Empty'
+PRE_FILTER_VALUE_EXISTS = 'Exists'
+PRE_FILTER_VALUE_NOT_EQUAL = 'Value Not Equal'
```

Comment:
Probably not applicable for NYS, but should these be translated?

Reply:
Yeah, they should but I decided not to because it seemed a bit complicated.

The current code does not translate them and also relies on the values matching the values in [this js file](https://github.com/dimagi/commcare-hq/blob/02b249c5b5b81cb7389b024a0c513b8fda2fea0c/corehq/apps/userreports/static/userreports/js/constants.js#L7). Also the same values are sent back from the front end and used to map to the [filter generating logic here](https://github.com/dimagi/commcare-hq/blob/02b249c5b5b81cb7389b024a0c513b8fda2fea0c/corehq/apps/userreports/reports/builder/forms.py#L239). Really there should be system-readable keys and human-readable values that are translated, but the current code combines them and I didn't feel it was appropriate to take on changing that in the scope of the work I've been doing.